autoload/quick_scope/lazy_print.vim

```diff
@@ -0,0 +1,9 @@
+" function to wait to print error messages
+function! quick_scope#lazy_print#err(message) abort
+  augroup quick_scope_lazy_print
+    autocmd!
+    " clear the augroup so that these lazy loaded error messages only execute
+    " once after starting
+  augroup END
+  echoerr 'quick_scope ' . a:message
```

Comment:
What do you think about using `echomsg` instead of `echoerr`? We don't really need the line number for the error message.

Reply:
I think it probably makes more sense to just use `echomsg`, we don't really have an error, but we want the user's attention.